migrations/2_deploy_contracts.js

```diff
@@ -18,6 +18,8 @@ const ManualApprovalTransferManagerFactory = artifacts.require("./ManualApproval
 const ManualApprovalTransferManagerLogic = artifacts.require("./ManualApprovalTransferManager.sol");
 const CappedSTOFactory = artifacts.require("./CappedSTOFactory.sol");
 const CappedSTOLogic = artifacts.require("./CappedSTO.sol");
+const POLYCappedSTOFactory = artifacts.require("./POLYCappedSTOFactory.sol");
+const POLYCappedSTOLogic = artifacts.require("./POLYCappedSTO.sol");
```

Comment:
Let's not deploy it in initial migration as this is an experimental module right now.

Reply:
done